src/pivot-table/components/cells/DimensionTitleCell.tsx

```diff
@@ -89,7 +89,7 @@ const DimensionTitleCell = ({
       isDimension={cell.isDim}
       style={{
         ...style,
-        ...getBorderStyle(true, isLastColumn, styleService.grid.border),
+        ...getBorderStyle(isLastRow, isLastColumn, styleService.grid.border),
```

Comment:
`getBorderStyle` no longer includes the `cellStyle`. You will need to importat that as `baseCellStyle`

Reply:
Done